telescope/src/main/java/com/mattprecious/telescope/TriggerProcessorListener.java

```diff
@@ -0,0 +1,11 @@
+package com.mattprecious.telescope;
+
+import android.support.annotation.Nullable;
+
+public abstract class TriggerProcessorListener {
+  public abstract void onTriggerReady(@Nullable ScreenshotMode screenshotMode);
```

Comment:
Passing `null` tells Telescope to use the currently set mode, but this also allows it to be overridden.
Looking at this, I'm thinking this is useless and should be removed.